src/style.css

```diff
@@ -1,3 +1,33 @@
+/* stylelint-disable font-family-name-quotes */
 body {
+  background-color: #858585;
+  font-family: 'Roboto', sans-serif;
   margin: 0;
 }
+
+.nav {
+  justify-content: flex-end;
+  font-style: 15px;
+  background-color: #ffff;
+  display: flex;
+  align-items: center;
+}
+
+.nav__link {
+  padding: 25px;
+  text-decoration: none;
+}
+
+.aimg {
+  position: absolute;
+  margin-top: 10px;
+  margin-left: 50px;
+}
```

Comment:
Avoid using absolute positioning for elements unless it's necessary for the layout. It can lead to less maintainable and unpredictable styling. Consider using flexbox, grid, or other layout techniques.